app/code/Magento/Checkout/view/frontend/web/js/view/summary/item/details.js

```diff
@@ -22,7 +22,7 @@ define([
         getNameUnsanitizedHtml: function (quoteItem) {
             var txt = document.createElement('textarea');
 
-            txt.innerHTML = quoteItem.name;
```

Comment:
Hello @Shivam7-1,

Please correct me if I am wrong, but here we are initializing the `txt.innerHTML` and in the next line the same has been sanitized using `escaper.escapeHtml(txt.value, this.allowedTags)`. Does this change is even required?

Reply:
Hii @engcom-Hotel Thanks For Review But I think yes Using textContent is more beneficial here as it automatically ensures text is safely handled without risking HTML injection, unlike innerHTML.

Reply:
Yes @Shivam7-1, I agree that using `textContent` is beneficial in order to prevent the HTML injection, but for the same we are already sanitizing the inputs here:

https://github.com/magento/magento2/blob/5184c0675bdec00396e2e555344d85d133d74de6/app/code/Magento/Checkout/view/frontend/web/js/view/summary/item/details.js#L27

Or you can provide us the manual testing scenario to test this PR change, it will help us with proceed further with this PR.